etna/ciim/client.py

```diff
@@ -8,15 +8,15 @@
 
 
 class KongClient:
-    def __init__(self, base_url, api_key):
+    def __init__(self, base_url, api_key, verify_certificates=True):
         self.base_url = base_url
         self.session = requests.Session()
         self.session.headers.update({"apikey": api_key})
+        self.session.verify = verify_certificates
 
     def fetch(self, **kwargs):
         kwargs["ref"] = kwargs.pop("reference_number", None)
```

Comment:
What if pop fails here ? and the statement not within the try.

Reply:
`kwargs` is adictionary containing keyword arguments for the function.

We're using `pop` here to fetch a value with the key `"start"` from the dictionary or return the default value `0`.

https://docs.python.org/3/library/stdtypes.html?highlight=pop#dict.pop

Reply:
I see since you are providing the default value None as a second argument.